tests/std/tests/Dev10_555491_complex_linker_errors/test.cpp

```diff
@@ -47,6 +47,87 @@ bool nearly_equal_partwise(const complex<T> actual, const complex<T> expected) {
     return nearly_equal(actual.real(), expected.real()) && nearly_equal(actual.imag(), expected.imag());
 }
 
+// Also test GH-2728 <complex>: Implementation divergence for division by zero
+void test_gh_2728() {
+    const complex<double> zero{0, 0};
+    {
+        const complex<double> test_one_one = complex<double>{1, 1} / zero;
+        assert(isinf(test_one_one.real()) && test_one_one.real() > 0);

```

Comment:
It's not a big deal here, but as a general rule, it's better to separately assert conditions with `assert(is_cute); assert(is_fluffy);` instead of `assert(is_cute && is_fluffy);`. That way, if the code ever encounters a cute Sphynx kitten, or an ugly fluffy kitten, then the assertion line will indicate exactly what went wrong.